presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -141,6 +153,20 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
         }
     }
 
+    public static DistributedQueryRunner createMaterializingQueryRunner(Iterable<TpchTable<?>> tables)
+            throws Exception
+    {
+        return createQueryRunner(
+                tables,
+                ImmutableMap.of(
+                        "query.partitioning-provider-catalog", "hive",
+                        "query.materialize-exchanges-strategy", "ALL",
+                        "query.hash-partition-count", "5",
```

Comment:
why not the classic number of `11`? This has more rounds for grouped execution :)

Reply:
No reason. Let me change that.